deployement/fix-mapping/FIX.md

```diff
@@ -0,0 +1,29 @@
+# Mappingservice fix for LifeCycle
+Mappingservice chokes in Harmonizations table copy.
+
+Sources:
+https://drive.google.com/drive/u/0/folders/1LgShQvLN8GHTLZv-xvQvGBbfoW7zN3W1
+
+## Prerequisites
+- Upload the new “HarmonizationsShadow” to production
+- Update mapping target entity to LifeCycle_HarmonizationsShadow 1x
+- Update mapping entity targets to LifeCycle_HarmonizationsShadow 22x
+
+## Steps to run the mapping
```

Comment:
add note that describes these steps need to me 'molgenis scheduled jobs' maybe add line line , for example

02:00 run remove att script
02:15  run att vars script 
....